.github/ISSUE_TEMPLATE/epic.yml

```diff
@@ -25,23 +25,75 @@ body:
       options:
         - label: I added a descriptive title
           required: true
-        - label: I searched open reports and couldn't find a duplicate
+        - label: I searched open issues and couldn't find a duplicate
           required: true
+
   - type: textarea
-    id: summary
+    id: what
     attributes:
-      label: Summary
+      label: What?
       description: >-
-        Define the highlevel objectives to be accomplished in this epic. Include the
-        bigger picture of what is changing and/or the user story for why the
-        changes are desired/necessary.
+        What feature or problem will be addressed in this epic?
+      placeholder: Please describe here.
     validations:
       required: true
+
   - type: textarea
+    id: why
     attributes:
-      label: Linked Issues & PRs
-      description: List all issues related to this epic.
+      label: Why?
+      description: >-
+        Why is the reported issue(s) a problem, or why is the proposed feature needed?
+        (Research and spike issues can be linked here.)
       value: |
-        - [ ] #
+        Please describe here and/or link to relevant supporting issues.
+        - [ ] <supporting issue from user, research, or spike>
```

Comment:
```suggestion
        - [ ] <supporting issue from user, research, or spike>
      placeholder: Please describe here and/or link to relevant supporting issues.
```

Reply:
Thanks, @kenodegard! I'm now using `placeholder` for "Please describe..." hints, and `value` for empty checkboxes.